### PR TITLE
CONTRIBUTING.md: link to "Faster reviews" document

### PR DESCRIPTION
**Pull request by andreynering**

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Comment by strk** [comment]
LGTM

**Comment by lunny** [comment]
This PR need @tboerger and @bkcsoft review.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -76,6 +76,19 @@ who makes the change even if an owners or a maintainer.  We use github's
 pull request workflow to do that and use [lgtm](http://lgtm.co) to ensure
 every PR is reviewed by at least 2 maintainers.
 
+Please, make the effort to make your pull request easy to review.
```
Comment (tboerger) [inline CONTRIBUTING.md]: Please try to make your pull request easy to review for us. Please read the "[How to get faster PR reviews]()" guide, it got useful...

Comment (andreynering) [reply CONTRIBUTING.md]: Updated

**Comment by bkcsoft** [comment]
LGTM

**Pull request merged**